src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/Documents.java

```diff
@@ -76,13 +75,12 @@ private static String getDocumentId(Map<String, Object> document) {
     }
 
 
-    static Map<String, Object> insertNewRecords(List<Map<String, Object>> exceptionDocuments,
-                                                List<Map<String, Object>> existingDocuments) {
-        ImmutableList<Object> documentList = ImmutableList.builder()
+    static List<Object> addNewRecords(List<Map<String, Object>> exceptionDocuments,
```

Comment:
perhaps `concatDocuments` or `joinDocuments` would be clearer.

Reply:
renamed the method